warrior/Actions/CiRegressionActions/ci_regression_actions.py

```diff
@@ -329,3 +329,99 @@ def delete_tmp_dir(self):
         """
         path = data_Utils.get_object_from_datarepository("parallel_exec_tmp_dir")
         return file_Utils.delFolder(path)
+
+    def check_kw_arg_type(self, str_value, int_value, float_value, bool_value,
+                          list_value, tuple_value, dict_value, file_value):
+        """Check the type of keyword arguments
+        :Argument:
+            1. str_value - expected to be string
+            2. int_value - expected to be int
+            3. float_value - expected to be float
+            4. bool_value - expected to be bool
+            5. list_value - expected to be list
+            6. tuple_value - expected to be tuple
+            7. dict_value - expected to be dict
+            8. file_value - expected to be file
+        """
+        status = True
+        err_msg = "{} is not an {} value but of type {}"
+        if type(str_value) is not str:
+            print_error(err_msg.format(str_value, "str", type(str_value)))
+            status = False
+        if type(int_value) is not int:
+            print_error(err_msg.format(int_value, "int", type(int_value)))
+            status = False
+        if type(float_value) is not float:
+            print_error(err_msg.format(float_value, "float", type(float_value)))
+            status = False
+        if type(bool_value) is not bool:
+            print_error(err_msg.format(bool_value, "bool", type(bool_value)))
+            status = False
+        if type(list_value) is not list:
+            print_error(err_msg.format(list_value, "list", type(list_value)))
+            status = False
+        if type(tuple_value) is not tuple:
+            print_error(err_msg.format(tuple_value, "tuple", type(tuple_value)))
+            status = False
+        if type(dict_value) is not dict:
+            print_error(err_msg.format(dict_value, "dict", type(dict_value)))
+            status = False
+        if type(file_value) is not file:
+            print_error(err_msg.format(file_value, "file", type(file_value)))
+            status = False
+        return status
+
+    def check_values_from_datafile(self, system_name, strvar, langs, states,
+                                   currencys, ramspace, configfile, intvar,
+                                   file_config):
+        """get the values from datafile
```

Comment:
This keyword description should be
Verify the datatype of the value read from the datafile using either the tag or wtag feature

Reply:
@navvrnd 
Looks like this comment get hidden but not addressed, can you update the description please?